google/cloud/spanner_v1/_helpers.py

```diff
@@ -196,6 +198,152 @@ def _datetime_to_rfc3339_nanoseconds(value):
     return "{}.{}Z".format(value.isoformat(sep="T", timespec="seconds"), nanos)
 
 
+@dataclass
+class Interval:
```

Comment:
Why this class is added in _helpers file. Shouldn't it be added in `data_types.py` file

Reply:
moved